app/views/evaluator_submission_assignments/_submission_row.html.erb

```diff
@@ -1,25 +1,32 @@
-<tr data-evaluator-id="<%= evaluator.id %>" data-evaluator-type="user">
+<tr data-evaluator-id="<%= evaluator.id %>" data-evaluator-type="user" class="border-y-2px border-primary" style="height: 4rem;">
   <td data-label="Submission ID" class="text-top text-bold">
-    <% if assignment.status == "recused" %>
-      <%= image_tag('images/usa-icons/error.svg', class: "usa-icon--size-3", alt: "Recused", style: "vertical-align: middle; margin-right: 5px;") %>
-    <% end %>
-    <%= assignment.submission.id %>
+    <div class="margin-bottom-1 mobile-lg:margin-bottom-3">
+      <%= link_to submission_path(assignment.submission), class: "text-primary" do %>
+        <span class="mobile-lg:display-none">Submission ID </span><%= assignment.submission.id %>
+      <% end %>
+    </div>
   </td>
-  <td data-label="Evaluation status">
-    <span class="text-top usa-tag <%= evaluation_status_color(assignment.evaluation_status) %>"><%= assignment.evaluation_status.to_s.titleize %></span>
+  <td data-label="Evaluation Status">
+    <span class="usa-tag padding-x-1 text-center width-15 <%= evaluation_status_color(assignment.evaluation_status) %>" style="min-width: 120px; display: inline-block;">
+      <%= assignment.evaluation_status.to_s.titleize %>
+    </span>
   </td>
-  <td data-label="Score" class="text-top text-normal"><%= assignment_display_score(assignment) %></td>
-  <td data-label="">
-    <div class="display-flex flex-justify-end">
-      <% if assignment.evaluation_status == :completed %>
-        <%= link_to "View Evaluation", revision_evaluation_path(assignment.evaluation), class: 'usa-button font-body-3xs margin-right-1' %>
-      <% end %>
-      <%= button_tag "Unassign", type: 'button', class: 'usa-button usa-button--outline font-body-3xs', data: { 
-        action: "click->unassign-evaluator-submission-modal#open",
-        assignment_id: assignment.id,
-        phase_id: @phase.id,
-        current_status: assignment.status
-      } %>
+  <td data-label="Score" class="text-top text-normal">
+    <%= assignment_display_score(assignment) %>
+    <% if assignment.evaluation_status == :completed %>
+      - <%= link_to "View Evaluation", revision_evaluation_path(assignment.evaluation), class: 'margin-left-2' %>
```

Comment:
I don't see the `margin-left-2` anywhere on figma, let's remove that
```suggestion
      - <%= link_to "View Evaluation", revision_evaluation_path(assignment.evaluation) %>
```

Reply:
note: the desktop layout doesn't have a "-" before the link... _please ignore that!_ score and link should appear the same on both layouts.